### DISCUSSION TITLE
Push notifications not appearing

### DISCUSSION THREAD

**Discussion by codyfcook** [community/community#141096] [Mobile] [answered]
I have to open the app to see them. I've turned on notifications in the iOS settings and for GitHub. I've also turned off working hours so that they should come through at all times. Any ideas? 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.181.0 (143346435)
Device: iPhone 14 Pro (iOS 18.0.1)
</details>

**Comment by Kithru** [comment]
Sometimes, GitHub’s notification service might be down. You can check GitHub’s status

**Reply by codyfcook** [reply]
Unfortunately it's been like this for months

**Comment by stevepopovich** [comment]
@codyfcook sorry you are having issues. Have you reinstalled the app?

**Reply by codyfcook** [reply]
Yes I did try that. 

Interestingly I got a notification for your mention of my username. (But I didn't get a notification for a reply in a thread in a private repo, so doesn't seem to be fixed)

**Reply by stevepopovich** [reply]
At this time, we don't support push notifications for replies in threads. These are the only things we support has far as push notifications go: 
![Screenshot_20241011_121731](https://github.com/user-attachments/assets/f45c3271-2a67-4448-bda5-cb7c1c777c9d)

**Reply by codyfcook** [reply, marked as answer]
Ahh gotcha. Thanks for letting me know.
